src/os/bluestore/Allocator.cc

```diff
@@ -26,13 +26,23 @@ class Allocator::SocketHook : public AdminSocketHook {
                                            "dump allocator free regions");
       if (r != 0)
         alloc = nullptr; //some collision, disable
+      if (alloc) {
+        r = admin_socket->register_command(("bluestore allocator score " + name).c_str(),
+                                           ("bluestore allocator score " + name).c_str(),
+                                           this,
+                                           "give measurement on allocator fragmentation quality (0-bad, 1-perfect)");
```

Comment:
does 'perfect' mean entirely fragmented? on an empty cluster this score is very close to 0

Reply:
@jdurgin Text was incorrect. Now fixed I hope. Please check.